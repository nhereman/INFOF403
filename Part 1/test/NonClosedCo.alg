co Non Closed Comment test co

begin
	read(a)			co We read a co
	read(b)			co We read b
	c:= (a+b)
	print(c)
end