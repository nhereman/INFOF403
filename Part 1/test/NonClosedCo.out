token: begin	lexical unit: BEG
token: read	lexical unit: READ
token: (	lexical unit: LEFT_PARENTHESIS
token: a	lexical unit: VARNAME
token: )	lexical unit: RIGHT_PARENTHESIS
token: read	lexical unit: READ
token: (	lexical unit: LEFT_PARENTHESIS
token: b	lexical unit: VARNAME
token: )	lexical unit: RIGHT_PARENTHESIS
Parse Error : File end with non closed comment
